browser-sign-in/OktaBrowserSignIn/WelcomeViewController.swift

```diff
@@ -80,3 +77,14 @@ extension UIViewController {
         }
     }
 }
+
+public extension UIColor {
+    /// Define two colors for both light and dark mode
+    static func setColor(lightMode: UIColor, darkMode: UIColor) -> UIColor {
```

Comment:
I'm not sure this is necessary. Shouldn't you be able to define that in the storyboard? In fact, the default should just "Do The Right Thing" I thought.

Reply:
I could change the label colour to use the system default colour. However It just looks Jenky in my opinion because the view background remains white but the label has a dark background colour.

Reply:
This is how it looks with the current changes. 
![Simulator Screen Shot - iPhone 13 - 2022-07-29 at 13 56 04](https://user-images.githubusercontent.com/108691051/181817434-f3b837be-695c-4f5e-ba31-9edfcad3288f.png)

Reply:
Thanks. I've updated this to use the system colour on the storyboard